### PR TITLE
Welcome email

### PR DESCRIPTION
**Pull request by drphilmarshall**
@abmantz What else do we need to say in this first communication?

### CODE REVIEW

File: doc/Welcome.email [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+Dear PHYS366 Class,
+
+Welcome to "Special Topics in Astrophysics: Statistical Methods"! Our first session will be on Tuesday January 9 at 3:00 PM, in Building 380 Room 381T (https://goo.gl/maps/7hxR6TjkuUE2).
```
Comment (abmantz) [inline doc/Welcome.email]: could add that this is on the ground floor of bldg 380

File: doc/Welcome.email [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+Before then, we'd like you to read through the "Information for Stanford Students" on the course GitHub repository at https://github.com/KIPAC/StatisticalMethods - that page explains how the course will work, and what you need to do to prepare for it.
+
+In particular, there's a bit of setup that you will need to do in order to be able to access and submit the PHYS366 homework this quarter. This week, we're assigning you the world's simplest homework problem: submitting your solution to it will show that you are ready for the course. Access to this "Week 0" homework (and all subsequent problem sheets), is via the private GitHub repository at https://github.com/drphilmarshall/PHYS366-Homework-2017 - **please reply with your GitHub username** so that we can grant you access while you read the general instructions.
```
Comment (abmantz) [inline doc/Welcome.email]: (you will need to make yourself an account if you don't already have one)

Comment (abmantz) [reply doc/Welcome.email]: Also note that, once the quarter gets going, we will primarily be using "issues" in this homework repository when we need to make announcements to the entire class - so make sure you are set up to "watch" the repository once you are granted access.

(The documentations says this, but it bears repeating.)

**Comment by drphilmarshall** [comment]
Great, thanks Adam. Sending now!

**Pull request merged**